client/js/components/App.js

```diff
@@ -27,10 +28,11 @@ class App extends Component {
     const { currentUser, actions } = this.props;
 
     return (
-      <div className="col-md-10 col-md-push-1">
-        <Header user={currentUser} onLogout={actions.logout} onLogoutSuccess={this.redirectToHomePage.bind(this)}/>
+      <section>
```

Comment:
We need to define some formatting rules and find tools to automate formatting and enforce the rules. My IDE by default removes the spaces you just added.